### PR TITLE
Cypress test. Rotate all images feature

### PR DESCRIPTION
**Pull request by dvkruchinin**
### Motivation and context
Cypress test. Rotate all images feature.

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2020 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

File: tests/cypress/integration/actions_tasks_objects/case_19_all_image_rotate_features.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    describe(`Testing case "${caseId}"`, () => {
+        it('Rotate an image (once clockwise, twice anticlockwise)', () => {
+            imageRotate('clockwise', 90);
+            imageRotate('anticlockwise', 0);
+            imageRotate('anticlockwise', 270);
+        });
+
+        it("Go to the next frame. It wasn't rotated.", () => {
+            cy.get('.cvat-player-next-button').click();
+            checkDegRotate(0);
```
Comment (bsekachev) [inline tests/cypress/integration/actions_tasks_objects/case_19_all_image_rotate_features.js:39]: Please check additionally that the frame has been changed

File: tests/cypress/integration/actions_tasks_objects/case_19_all_image_rotate_features.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+        it('Rotate current frame 180 deg.', () => {
+            imageRotate('clockwise', 90);
+            imageRotate('clockwise', 180);
+        });
+
+        it('Go to the previous and to the next frame. They are also rotated 180 deg.', () => {
+            cy.get('.cvat-player-previous-button').click();
+            checkDegRotate(180);
+            cy.get('.cvat-player-next-button').click();
+            checkDegRotate(180);
```
Comment (bsekachev) [inline tests/cypress/integration/actions_tasks_objects/case_19_all_image_rotate_features.js:57]: Please check additionally that the frame has been changed

File: tests/cypress/integration/actions_tasks_objects/case_19_all_image_rotate_features.js (hunk 1 above)
Comment (dvkruchinin) [reply tests/cypress/integration/actions_tasks_objects/case_19_all_image_rotate_features.js:46]: Added a separate function for this check. The check is performed by the frame number in the field ```.cvat-player-frame-selector```.

File: tests/cypress/integration/actions_tasks_objects/case_19_all_image_rotate_features.js (hunk 2 above)
Comment (dvkruchinin) [reply tests/cypress/integration/actions_tasks_objects/case_19_all_image_rotate_features.js]: Done.

**Review by bsekachev — APPROVED** [review]

**Pull request merged**